runtime/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -1,12 +1,13 @@
 /*
  * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
+@file:Suppress("EXPERIMENTAL_API_USAGE")
```

Comment:
What's experimental in here?

Reply:
`@UnstableDefault`. It will be cleaned out along with ctor changes

Reply:
Why not `@OptIn(UnstableDefault)` then?

Reply:
This one was suggested by the quickfix first :)